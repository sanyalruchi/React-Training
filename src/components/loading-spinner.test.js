import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';

import LoadingSpinner from './LoadingSpinner';

describe('LoadingSpinner Component', () => {
    it('should render correctly', () => {
        const wrapper = shallow(<LoadingSpinner/>);
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    })
})