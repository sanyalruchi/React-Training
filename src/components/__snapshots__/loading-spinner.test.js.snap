// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner Component should render correctly 1`] = `
<div>
  Loading....
</div>
`;
